app/views/govuk_publishing_components/components/docs/chart.yml

```diff
@@ -0,0 +1,174 @@
+name: "Chart"
+description: "The chart component presents a data chart and a tabular version of the same data"
+part_of_admin_layout: true
+body: |
+  This component takes a set of data and presents it as a line graph and a table with one or more rows and lines.
+  The chart relies upon chartkick and renders using JavaScript, so the table is provided as a fallback for a lack of JavaScript,
+  an accessible view on the data for screenreaders, and a simple view of the raw data for all users.
+
+  The `overview` option can be used to provide a explanation for screen reader users of what the graph shows.
```

Comment:
Sorry really minor one! I think 'provide a explanation' should be 'provide an explanation'.

Reply:
Good spot, have fixed 👍